applications/StructuralMechanicsApplication/custom_processes/set_cartesian_local_axes_process.cpp

```diff
@@ -0,0 +1,80 @@
+// KRATOS  ___|  |                   |                   |
```

Comment:
in my opinion it is much clearer to have smaller processes that each do something different, CARTESIAN, CYLINDRICAL, etc than one doing a lot of things...

this is why i proposed Alejandro to split it in multiple simple processes rather than having a single complex one (with a more complex user input)

Reply:
ok